### ISSUE TITLE
Zuluhed the Whacked (BB #1179)

### ISSUE THREAD

**Issue by OregonAsari** [OregonCore/OregonCore#1179]
This issue was migrated from bitbucket.
**Original Reporter:** AlphafoxBHN
**Original Date:** 12.12.2015 19:28:03 GMT+0000
**Original Priority:** major
**Original Type:** bug
**Original State:** closed
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/1179

<hr>

Zuluhead Spawn on a wrong location. He Spawn on the left tower inside  the drake rider

http://www.wowhead.com/quest=10866/zuluhed-the-whacked

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 25.12.2015 01:50:17 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/1179/#comment-24313637

<hr>

Does this fix it?

```
#!sql

-- Fix spawn position for Zuluhed The Whacked
UPDATE `quest_start_scripts` SET `x`=-4204.937,`y`=316.3974,`z`=122.5078,`o`=1.308997 WHERE id=10866;

```

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 25.12.2015 18:06:41 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/1179/#comment-24320610

<hr>

[DB/Misc] Fix a leveling exploit.
          Fix Nazans inhabittype. Closes #1175
          Fix spawn position for Zuluhed the Whacked. Closes #1179
          Add script support for Zuluhed the Whacked.

→ <<cset 7174908c8251>>
